src/situations/inventaire/vues/contenu.js

```diff
@@ -21,26 +22,40 @@ class VueContenu {
       }, DELAY_FERMETURE_CONTENANT_MILLISEC);
       event.stopPropagation();
     });
-    this.element = document.createElement('div');
+    this.element = document.createElement('img');
     this.element.classList.add('contenu', 'fermer');
     pointInsertion.appendChild(this.element);
-
-    this.elementInterieur = document.createElement('div');
-    this.elementInterieur.classList.add('interieur');
-    this.element.appendChild(this.elementInterieur);
   }
 
   position (position, dimensionFermee, dimensionOuverte) {
-    return position + dimensionFermee / 2 - dimensionOuverte / 2;
+    var positionCalculee = position + dimensionFermee / 2 - dimensionOuverte / 2;
+    positionCalculee = Math.max(positionCalculee, 0);
+    positionCalculee = Math.min(positionCalculee, 100 - dimensionOuverte);
+    return positionCalculee;
+  }
+
+  calculeDimensionsAdaptees (contenant) {
```

Comment:
J'aurais bien nommé cette fonction calculeHauteurAdaptee à la quelle j'aurais passé la largeur (et aussi le contenant puisqu'on en a besoins).

Reply:
Et j'aurais bien retourné la hauteur du coup, plutôt que de l'assigner à this.dimensions.hauteur

Reply:
ok je l'ai fait